src/Http/Routing/test/FunctionalTests/EndpointRoutingSampleTest.cs

```diff
@@ -210,6 +210,19 @@ public async Task GeneratesLink_ToEndpointWithDoubleAsteriskCatchAllParameter_En
             Assert.Equal("Link: /WithDoubleAsteriskCatchAll/a/b%20b1/c%20c1", actualContent);
         }
 
+        [Fact]
+        public async Task MapGet_HasConventionMetadata()
+        {
+            // Arrange & Act
+            var response = await _client.GetAsync("/convention");
+
+            // Assert
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+
+            var actualContent = await response.Content.ReadAsStringAsync();
+            Assert.Equal("True", actualContent);
```

Comment:
Think about changing this to some text like `"Has custom metadata: True"`. I just imagine (if this managed to fail) how someone would even start to investigate without knowing exactly what code is called on the sever.